### PR TITLE
FIX: Split multiple apps and bins by space

### PR DESCRIPTION
**Pull request by felixhoffmnn**
## Description

> See #1083

Based on the docs, it should be possible to provide a list to `exported_bins`. This PR addresses this issue and extends the functionality also to the `exported_apps`.

**Note:** This is my first PR to this project. Please don't hesitate to test and modify it to your standards.

## Testing

By this fix, I was able to run the example shown in the referred issue.

### CODE REVIEW

**Comment by felixhoffmnn** [comment]
I think it would also be good to add an example for exporting multiple apps.

File: distrobox-assemble [hunk 1]
```diff
@@ -370,15 +370,27 @@ run_distrobox() (
...
 		IFS="¤"
-		for app in ${exported_apps}; do
-			app="$(echo "${app}" | tr -d '[:space:]')"
-			"${distrobox_path}"/distrobox enter "${name}" -- distrobox-export --app "${app}"
+		for apps in ${exported_apps}; do
+			# Split the string by spaces
+			IFS=" "
+			for app in ${apps}; do
+				# Remove spaces
+				app="$(echo "${app}" | tr -d '[:space:]')"
```
Comment (dkrulev-sdc) [inline distrobox-assemble:378]: the script involves splitting a string by spaces and then trimming spaces from each resulting element. However, I'm curious if the trimming step is actually necessary.

I tried the following code in shell, results are identical
```sh
apps="1   2    3  4 5 6"
IFS=" "
echo with trimming
for app in ${apps}; do
        app="$(echo "${app}" | tr -d '[:space:]')"
        echo "|$app|"
done
echo without trimming
for app in ${apps}; do
        echo "|$app|"
done
```

Comment (felixhoffmnn) [reply distrobox-assemble]: You're right. I removed the `tr -d '[:space:]')`.

**Comment by 89luca89** [comment]
Thanks a lot @felixhoffmnn this will Fix #1083

**Pull request merged**